src/component/Shopping/shopping.scss

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
please remove this file.

for global style, please move it into a `_reset.scss` and import into `index.js`